packages/playground/src/utils/validators.ts

```diff
@@ -777,3 +777,11 @@ export async function isAvailableName(grid: GridClient, name: string) {
     return { message: "Name is already taken." };
   }
 }
+
+export function validateSmtp(v: string) {
+  const emailValidation = isEmail("Please provide a valid email address.")(v);
+  const username = /[!@#$%^&*()_+\-={}:<>?,./]/.test(v);
```

Comment:
why are you using regex here?

Reply:
What are the requirements of the username?

Reply:
no requirement just string, just added regex to avoid symbols so we can validate emails too in the same input